stripe/build.gradle

```diff
@@ -6,12 +6,13 @@ configurations {
 
 dependencies {
     compile 'com.android.support:support-annotations:25.1.0'
+    compile 'com.android.support:support-v4:25.1.0'
```

Comment:
If we wanted to, we could write this groovy code to keep the `com.android.support` versions synchronized:

```groovy
def v_android_support = "25.1.0"
// ...
dependencies {
  compile 'com.android.support:support-annotations:' + v_android_support
  compile 'com.android.support:support-v4:' + v_android_support
  javadocDeps 'com.android.support:support-annotations:' + v_android_support
  // ... etc
}
```

On that note: we refer to `buildToolsVersion '25.0.0'` below, but this is version 25.1.0. These are suspiciously similar versions, but I don't know enough about them. Should they be the same version?